packages/reactive_ble_mobile/android/src/main/kotlin/com/signify/hue/flutterreactiveble/model/BondingMode.kt

```diff
@@ -0,0 +1,9 @@
+package com.signify.hue.flutterreactiveble.model
+
+import android.content.Context
+
+enum class BondingMode(val code : Int) {
```

Comment:
I think we can get rid of this right when we go for a separate call to create bond?

Reply:
I removed `BondingMode`, but introduced a new `BondingStatus` enum. The enum reflects the Android states (none, bonding, bonded) and could be used in the future for an observing component as well. Point is there's a need to know the outcome of the bonding (and "failed"/not bonded may not always be a problem for the calling app). Therefore we return the actual value here.